### ISSUE TITLE
Page title for SimpleReport support is not descriptive enough

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4041]
## Area or Component
simplereport.gov/support

## Quick Summary
Title is "SimpleReport", but should follow the pattern elsewhere on the static site to "SimpleReport support" or "Support | SimpleReport"

## Accessibility Violation
Page <title> does not identify purpose of page: https://cdc-devtools.dequecloud.com/rule-help/en/title-not-meaningful

## Action Requested
Make the title of the support page more meaningful.

## Original Deque Issue
https://docs.google.com/spreadsheets/d/16udNpwdJ8sltAULnmSIGgkWVQsfuFvcoymP0-s_QwfE/edit#gid=1832428693&range=14:14
